Purchases/Purchasing/StoreKitAbstractions/StoreProductDiscount.swift

```diff
@@ -116,9 +107,34 @@ public extension StoreProductDiscount {
 
 }
 
+extension StoreProductDiscount {
+
+    /// Used to represent ``StoreProductDiscount/id``. Not for public use.
+    public struct Data: Hashable {
```

Comment:
Note that only the type is `public` because it needs to be exposed for the id, but it has no public initializers or members.